AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/core/handler/CheckedChangeHandler.java

```diff
@@ -50,9 +53,9 @@ public void validate(Element element, ElementValidation validation) {
 		ExecutableElement executableElement = (ExecutableElement) element;
 		validatorHelper.returnTypeIsVoid(executableElement, validation);
 
-		validatorHelper.param.anyOrder() //
-				.extendsType(CanonicalNameConstants.COMPOUND_BUTTON).optional() //
-				.primitiveOrWrapper(TypeKind.BOOLEAN).optional() //
+		validatorHelper.param.inOrder() //
+				.extendsAnyOfTypes(CanonicalNameConstants.COMPOUND_BUTTON, CanonicalNameConstants.RADIO_GROUP).optional() //
+				.anyOfPrimitiveOrWrapper(TypeKind.BOOLEAN, TypeKind.INT).optional() //
```

Comment:
For cases like this it would be awesome if our `ValidatorParameterHelper` would support some kind of `or`.

e.g.
```
validatorHelper.param
	.anyOrder()
	.extendsType(CanonicalNameConstants.COMPOUND_BUTTON).optional()
	.primitiveOrWrapper(TypeKind.BOOLEAN).optional()

	or()

	.anyOrder()
	.extendsType(CanonicalNameConstants.RADIO_GROUP).optional()
	.primitiveOrWrapper(TypeKind.INT).optional()

	.validate(executableElement, validation);
```